Alignment/include/ActsAlignment/Kernel/Alignment.ipp

```diff
@@ -6,6 +6,12 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+#pragma once
+
+// clang-format off
+#include "ActsAlignment/Kernel/Alignment.hpp"
+// clang-format on
```

Comment:
I am not 100% sure, but I remember it to be enough to just separate `#include`-blocks by an empty line or comment to avoid resorting.

Reply:
sadly it resorts it automatically for `.ipp`. maybe we can tell `clang-format` to treat `.ipp` like `.cpp`